src/fs.zig

```diff
@@ -203,22 +203,13 @@ pub const FileSystem = struct {
         // }
 
         pub fn addEntry(dir: *DirEntry, entry: std.fs.IterableDir.Entry, allocator: std.mem.Allocator, comptime Iterator: type, iterator: Iterator) !void {
-            var _kind: Entry.Kind = undefined;
```

Comment:
you can tell this is some of the earliest code in bun because i did not know zig very well